### DISCUSSION TITLE
[BUG] Connect-pnponline -S is not working any more

### DISCUSSION THREAD

**Discussion by gitulm** [pnp/powershell#4426] [General]
I use the following URL for various scripts
Connect-PnPOnline -S -Url https://xxx-admin.sharepoint.com -ClientID xxxxx -Credentials $spoCredential
This hasn't worked for a few days. I get the following error message:
"Parameter set cannot be resolved using the specified named parameters. One or more parameters issued cannot be used together or an insufficient number of parameters were provided."

If I use the URL without -S (Connect-PnPOnline -Url https://xxx-admin.sharepoint.com -ClientID xxxxx -Credentials $spoCredential), then in my script at the point:
Set-PnPUserProfileProperty -Account $User -PropertyName $SPOPropertyName -Values ​​$ADUserPropertyValue
I get the error:
This process requires that you manage your own data or have administrator rights

My account spoadmin is a Sharepoint admin.
I use Powershell 7.4.5 with PnP Powershell 2.12.0

How can I solve my problem?

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows

**Comment by ankeetmistry** [comment]
Same issue i am facing? any help would be highly appreciated
Error i receive:
Set-PnPTenantSite: Unable to connect to the SharePoint Online Admin Center at 'https://contoso-admin.sharepoint.com' to run this cmdlet. If this URL is incorrect for your tenant, you can pass in the correct Admin Center URL using Connect-PnPOnline -TenantAdminUrl. If you are using Privileged Identity Management (PIM) on your tenant, please ensure you have activated at least the SharePoint Administrator role and allowed some time for it to activate. Error message: Token request failed.

**Comment by gitulm** [comment]
The problem no longer occurs if I don't use the ClientID parameter:
Connect-PnPOnline -Url https://xxx-admin.sharepoint.com/ -Credentials $spoCredential
